seed/php-sdk/pagination/no-custom-config/README.md

```diff
@@ -68,6 +68,41 @@ try {
 }
 ```
 
+## Pagination
+
+List endpoints return a `Pager<T>` which lets you loop over all items and the SDK will automatically make multiple HTTP requests for you.
+
+```php
+<?php
+
+namespace Example;
+
+use Seed\SeedClient;
+use Seed\Complex\Types\SearchRequest;
+use Seed\Complex\Types\StartingAfterPaging;
+use Seed\Complex\Types\SingleFilterSearchRequest;
+use Seed\Complex\Types\SingleFilterSearchRequestOperator;
+
+$client = new SeedClient(
+    token: '<token>',
+);
+$client->complex->search(
+    'index',
+    new SearchRequest([
+        'pagination' => new StartingAfterPaging([
+            'perPage' => 1,
+            'startingAfter' => 'starting_after',
+        ]),
+        'query' => new SingleFilterSearchRequest([
+            'field' => 'field',
+            'operator' => SingleFilterSearchRequestOperator::Equals->value,
+            'value' => 'value',
+        ]),
+    ]),
+);
+
+```
```

Comment:
@stash-rl the unique thing about pagination snippets is that you want to show how to handle the response: 
```
foreach ($payments as $payment) {
    echo sprintf(
        "payment: ID: %s Created at: %s, Updated at: %s\n",
        $payment->getId(),
        $payment->getCreatedAt(),
        $payment->getUpdatedAt(),
    );
}
```